### ISSUE TITLE
Dynamic Refinement Network for Oriented and Densely Packed Object Detection

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#68]
# TL;DR
It is difficult to detect objects that are dense and rotated with respect to the image. To overcome this difficulty, they proposed a model based on CenterNet, which regresses the object center without using BoundingBox, to detect along the line of the object to be detected using Deformable Conv, and DRH, which dynamically responds to each sample by and generating filters.
![Feature selection](https://user-images.githubusercontent.com/24524018/92360834-6b55ea80-f128-11ea-81e8-ab58498ca5af.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2005.09973

# Submission Dates(yyyy/mm/dd)
2020/5/20

# Authors and institutions
Xingjia Pan, Yuqiang Ren, Kekai Sheng, Weiming Dong, Haolei Yuan, Xiaowei Guo, Chongyang Ma, Changsheng Xu
- NLPR, Institute of Automation, CAS 
- School of Artificial Intelligence, UCAS 
- Youtu Lab, Tencent 
- CASIA-LLVision Joint Lab 
- Y-Tech, Kuaishou Technology

# Methods

# Results

# Comments
CVPR oral
